cmd/osquery-perf/agent.go

```diff
@@ -469,6 +469,15 @@ func (a *agent) munkiInfo() []map[string]string {
 	}
 }
 
+func (a *agent) googleChromeProfiles() []map[string]string {
+	count := rand.Intn(5) // return between 0 and 4 emails
+	result := make([]map[string]string, count)
+	for i := range result {
+		result[i] = map[string]string{"email": fmt.Sprintf("user%d@example.com", i)}
```

Comment:
Searching for `user1@example.com` will return a lot of hosts. I suppose it's not incorrect but I wonder if a more "realistic" scenario would be to use the random host name in the email? Or at least in one of them, so that we have both cases, one where the search returns a lot of hosts, and others where it returns exactly one.

Reply:
Yeah I wasn't sure what we'd want to test with that exactly - to load-test the ingestion, this is probably close enough, as the churn is not too big for any given host (maybe a few emails less or more, but stable sometimes). What I can do is if the rand returns 4, set the domain part of the last email to something with a random number in it, so it matches few(er) hosts? I'll go with that for now, let me know if that's good enough.

Reply:
Sounds good.